examples/integration-tests/test-node.sh

```diff
@@ -8,6 +8,22 @@ node ./app.js &
 APP_PID=$!
 cd -
 sleep 5
-curl -v -s --show-error -X POST -d @requests/paymentOrderRequest_v1.json -H "Content-Type: application/json" -H "x-payex-sample-apikey: $API_KEY" -H "x-payex-sample-access-token: doot_doot" localhost:8080/paymentorders | jq .
+result=$(curl -v -s --show-error -X POST -d @requests/paymentOrderRequest_v1.json -H "Content-Type: application/json" -H "x-payex-sample-apikey: $API_KEY" -H "x-payex-sample-access-token: doot_doot" localhost:8080/paymentorders)
```

Comment:
I would prefer if this test was written in JavaScript, as a Node.js test in Mocha, Chai, Jest, or similar.